packages/preview-middleware/test/unit/base/__snapshots__/flp.test.ts.snap

```diff
@@ -258,7 +259,39 @@ exports[`FlpSandbox router test/flp.html 1`] = `
 </html>"
 `;
 
-exports[`FlpSandbox router test/flp.html?fiori-tools-rta-mode=forAdaptation 1`] = `
+exports[`FlpSandbox router rta with developerMode=true 1`] = `
+"<!DOCTYPE html>
+<html lang=\\"en\\">
+    <head>
+        <meta charset=\\"utf-8\\" />
+        <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1\\" />
+        <meta name=\\"theme-color\\" content=\\"#000000\\" />
+        <meta name=\\"description\\" content=\\"Web page for rendering Open UX Tools Control Property Editor\\" />
+        <link rel=\\"shortcut icon\\" type=\\"image/x-icon\\" href=\\"./editor/favicon.ico\\" />
+        <link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"./editor/app.css\\" />
+        <title>Control Property Editor</title>
+        <style>
+            body, html {
+                margin: 0;
+                padding: 0;
+                height: 100%;
+                overflow: hidden;
+            }
+            iframe {
+                width: 100%;
+                height: 100%;
+                border: none;
+            }
+        </style>
+    </head>
+    <body>
+        <iframe src=\\"/my/editor.html.inner.html?sap-ui-xx-viewCache=false&fiori-tools-rta-mode=forAdaptation&sap-ui-rta-skip-flex-validation=true&sap-ui-xx-condense-changes=true#app-preview\\"></iframe>
```

Comment:
Having the `html` extension in the middle of name `editor.html.inner.html` looks a bit odd.

Reply:
I know but do we really care? I can fix it, but I didn't see a reason to make the url nicer.

Reply:
It's not really visible so I suppose its ok.